src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/test/java/org/openkilda/wfm/topology/flowmonitoring/service/FlowCacheServiceTest.java

```diff
@@ -197,6 +220,64 @@ public void serviceActivationDeactivationAndReactivation() {
         assertFalse(service.flowStatesIsEmpty());
     }
 
+    @Test
+    public void changeHaFlowPathInCache() {
```

Comment:
this is very difficult to follow what is going on in this test. I suggest to split it into methods with semantic given/when/then and descriptive method names to understand better what are preparation steps

Reply:
This test is consistent with other tests in the class.